docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,239 @@
+
+```csharp
+        protected override void OnLowMemory(LowMemoryEventArgs e)
+        {
+            base.OnLowMemory(e);
+        }
+```
+
+`OnLowBattery()` is used to take necessary actions in low battery situations.
+Save data in the main memory to a persistent memory or storage, to avoid data loss in case the power shut down completely. Stop heavy CPU consumption or power consumption activities to save the remaining power:
+
+```csharp
+        protected override void OnLowBattery(LowBatteryEventArgs e)
+        {
+            base.OnLowBattery(e);
+        }
+```
+
+`OnLocaleChanged()` and `OnRegionFormatChanged()` are invoked, when region or system language is changed:
+
+```csharp
+        protected override void OnLocaleChanged(LocaleChangedEventArgs e)
+        {
+            base.OnLocaleChanged(e);
+        }
+
+        protected override void OnRegionFormatChanged(RegionFormatChangedEventArgs e)
+        {
+            base.OnRegionFormatChanged(e);
+        }
+```
+
+```csharp
+        static void Main(string[] args)
+        {
+            App app = new App();
+            app.Run(args);
+        }
+    }
+}
+```
+
+## Application attributes
+Describe your service application attributes in the manifest file. The attributes determine the application behavior. The following code example illustrates how you can define the attributes:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns="http://tizen.org/ns/packages" api-version="6" package="org.tizen.example.servicesample" version="1.0.0">
+  <profile name="common" />
+  <service-application appid="org.tizen.example.servicesample"
+					exec="servicesample.dll"
+					type="dotnet"
+					multiple="false"
+					taskmanage="false"
+					nodisplay="true"
+                    api-version="8">
+    <label>servicesample</label>
+    <icon>servicesample.png</icon>
+  </service-application>
+</manifest>
+```
+
+- `auto-restart`
+
+  If set to `true`, the application restarts whenever it terminates abnormally. If the application is running, it is launched after installing or updating the package.
+
+  >[!NOTE]
+  > This attribute is not supported on Tizen wearable devices. Since Tizen 2.4, this attribute is not supported on all Tizen devices. Because of this, the `auto-restart` attribute used in a lower API version package than 2.4 is ignored on devices with the Tizen platform version 2.4 and higher.
+
+- `on-boot`
+
+  If set to `true`, the application launches on boot time, and after installing or updating the package. The application does not start, if this attribute is removed after updating the package.
+
+  >[!NOTE]
+  > This attribute is not supported on Tizen wearable devices. Since Tizen 2.4, this attribute is not supported on all Tizen devices. Because of this, the `on-boot` attribute used in a lower API version package than 2.4 is ignored on devices with the Tizen platform version 2.4 and higher.
```

Comment:
Note in line 223: Same issue here, these are about Tizen version before Tizen 2.4, but Tizen .NET started from Tizen 4.0, so I'm not sure if these notes are used to .NET as well. Please consider revising this.